hadoop-ozone/ozone-manager/src/test/java/org/apache/hadoop/ozone/om/TestBucketManagerImpl.java

```diff
@@ -57,7 +64,18 @@
   @Rule
   public TemporaryFolder folder = new TemporaryFolder();
 
-  private OzoneConfiguration createNewTestPath() throws IOException {
+  private OmTestManagers omTestManagers;
+  private OzoneManagerProtocol writeClient;
+  private OzoneManager om;
+
+  @After
+  public void cleanup() throws Exception {
+    om = omTestManagers.getOzoneManager();
+    om.stop();
+  }
+
+  private OzoneConfiguration createNewTestPath() throws IOException,
+          AuthenticationException {
```

Comment:
Does this really need the AuthenticationException?

Reply:
That's actually not needed, I will remove it. I declared inside the method omTestManagers but when I removed the field, I forgot about it.